src/scrabBag.js

```diff
@@ -0,0 +1,21 @@
+export default function scrabBag(theInput, tileDistro = {}) {
+  const inputArray = []
+
+  if (tileDistro === null) {
+    return 'cannot check null'
+  }
+  if (tileDistro.constructor !== Object || tileDistro.constructor === Array) {
+    return 'please use an object to create the distribution of letter tiles'
+  }
+  for (i = 0; i < theInput.length; i++) {
```

Comment:
Instead of using a for loop, you could call `.split('')` on a string to convert it into an array of characters:
```js
const array = theInput.split( '' )
```